frontend/src/components/Item/RelatedItems.vue

```diff
@@ -71,36 +67,37 @@ const props = withDefaults(
 );
 
 const relatedItems = ref<BaseItemDto[]>([]);
-const loading = ref(true);
+const loading = ref(false);
```

Comment:
It doesn't matter in practice, since the loading state will immediately be flipped to on, but just for consistency. All the other components with a loading property start with this `false` (actually I messed up ImageSearch when I re-wrote it, I'll update that too though.

I don't have an opinion for whether defaulting true or false is more correct, so I'm also happy to update all components using a `loading` variable to initialize to `true` if you prefer that more.

Reply:
@aweebs I don't recall any place where it's used with it initialized to false, but that doesn't make much sense imo, so ofc I'm all in for changing the defaults to true and avoid the extra assignment (and possibly DOM re-renders)

Reply:
Went through and you're correct, the others uses should be false to start off. With this one it should be true since the first mount action should always be a load (unless the prop item.Id isn't defined on first load, but that's probably enough of an edge case not to worry about). I'll update this!